packages/debug/vite.config.mjs

```diff
@@ -0,0 +1,17 @@
+import replace from '@rollup/plugin-replace';
```

Comment:
This file could use some eyes. It's not currently being used by the debug module, but could be useful for `vite` tests.

Reply:
debug does not have tests. We can just remove this.